modules/image-loading-optimization/assets/js/preview/index.js

```diff
@@ -0,0 +1,85 @@
+import { onLCP } from './web-vitals';
```

Comment:
Currently, `web-vitals.js` is placed in same folder. Couldn't figure out why direct import is not working.

cc: @joemcgill